test/pull.test.js

```diff
@@ -0,0 +1,14 @@
+const { pull } = require("../lib");
+
+describe("pull", () => {
+  test("pull [2, 4, 7] from [1, 2, 4, 6, 7] should equal to [1, 6]", () => {
+
+    expect(pull([1, 2, 4, 6, 7], [2, 4, 7])).toEqual([1, 6]);
+  });
+
+  test("pull 23, 7 from [23, 10, 2, 7] should equal to [23, 10]", () => {
+
+    expect(pull([23, 10, 2, 7], 23, 7)).toEqual([23, 10]);
+  });
+
```

Comment:
What should happen if the array has Complex objects like arrays or objects? or if the array is empty or if it isn't an array?

Reply:
@ktilcu I added more test cases for those